guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -26,18 +26,24 @@ When you set up the compute resource and images for VMware provisioning in {Proj
 * VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
 * The virtual machine builds, `cloud-init` is invoked and calls back {Project} on port `80`, which then redirects to `443`
 
-.Port and Firewall Requirements
+.Prerequisites
+* Configure port and firewall settings to open any necessary connections.
 Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}.
-Ensure that you configure port and firewall settings to open any necessary connections.
-
 ifndef::orcharhino[]
-For more information about port and firewall requirements, see {InstallingServerDocURL}satellite-ports-and-firewalls-requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _Installing {SmartProxyServer}_.
+For more information about port and firewall requirements, see {InstallingServerDocURL}satellite-ports-and-firewalls-requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _{InstallingSmartProxyDocTitle}_.
```

Comment:
I cannot find the anchor for the first link "-ports-and-firewalls-requirements_"; only the second one for capsule: `guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc:[id="capsule-ports-and-firewalls-requirements_{context}"]`.